dynawo/sources/Solvers/SolverIDA/DYNSolverIDA.cpp

```diff
@@ -814,6 +799,13 @@ SolverIDA::getRootsFound() const {
 
   return rootsFound;
 }
+#endif
+
+double SolverIDA::getTimeStep() const {
+  double hused;
+  IDAGetLastStep(IDAMem_, &hused);
+  return hused;
+}
```

Comment:
I think this is not used at the moment (except if I miss it).
I don't know what you had in mind when you added it but if there is a good reason to keep it, then we should use it in the getLastConf() method instead of retrieving it twice from IDA.

Reply:
getTimeStep is now a mandatory method of Solver base class so I had to implement it for IDA. I agree that it was not used but I now use it in getLastConf. In SolverSIM it is used to avoid having to pass along h (same was done for t) to the algebraic solver, better retrieve it from an API and be sure it is the right value.

Reply:
Sure, fine.